doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/project-storage-version-build-options-ref.adoc

```diff
@@ -0,0 +1,160 @@
+[id='project-storage-version-build-options-ref_{context}']
+= Project storage and build options with {PRODUCT}
+
+As you develop a {PRODUCT} project, you need to be able to track the versions of your project with a version-controlled repository, manage your project assets in a stable environment, and build your project for testing and deployment. You can use {CENTRAL} for all of these tasks, or use a combination of {CENTRAL} and external tools and repositories. {PRODUCT} supports Git repositories for project version control, Apache Maven for project management, and a variety of Maven-based, Java-based, or custom-tool-based build options.
+
+The following options are the main methods for {PRODUCT} project versioning, storage, and building:
+
+.Project version control options (Git)
+[cols="25%,50%,25%", options="header"]
+|===
+|Versioning option
+|Description
+|Documentation
+
+|{CENTRAL} Git VFS
+|{CENTRAL} contains a built-in Git Virtual File System (VFS) that stores all processes, rules, and other artifacts that you create in the authoring environment. Git is a distributed version control system that implements revisions as commit objects. When you commit your changes into a repository, a new commit object in the Git repository is created. When you create a project in {CENTRAL}, the project is added to the Git repository connected to {CENTRAL}.
+|NA
+
+|External Git repository
+|If you have {PRODUCT} projects in Git repositories outside of {CENTRAL}, you can import them into {PRODUCT} spaces and use Git hooks to synchronize the internal and external Git repositories.
+a|
+ifdef::DM,PAM[]
+{URL_MANAGING_PROJECTS}#git-import-project[_{MANAGING_PROJECTS}_]
+endif::[]
+ifdef::DROOLS,JBPM,OP[]
+NA
+endif::[]
+|===
+
+.Project management options (Maven)
+[cols="25%,50%,25%", options="header"]
+|===
+|Management option
+|Description
+|Documentation
+
+|{CENTRAL} Maven repository
+|{CENTRAL} contains a built-in Maven repository that organizes and builds project assets that you create in the authoring environment. Maven is a distributed build-automation tool that uses repositories to store Java libraries, plug-ins, and other build artifacts. When building projects and archetypes, Maven dynamically retrieves Java libraries and Maven plug-ins from local or remote repositories to promote shared dependencies across projects.
```

Comment:
i think it needs to highlight this is not recommended for production environment. An External Maven repository should be the main choice for real world prod env

Reply:
Done. Added this note. Work for you?

```
NOTE: For a production environment, consider using an external Maven repository configured with {CENTRAL}.
```

Reply:
+1